### ISSUE TITLE
Undefined symbol - glVertexPointer

### ISSUE THREAD

**Issue by sljunkie** [yquake2/yquake2#221]
Yamagui 7.01, Manjaro / Arch Linux (AUR package here: https://aur.archlinux.org/packages/yamagi-quake2/)

```
$ yamagi-quake2 

Yamagi Quake II v7.01
=====================

Client build options:
 + SDL2
 - CD audio
 + OGG/Vorbis
 + OpenAL audio
 + Zip file support
Platform: Linux
Architecture: x86_64
Byte ordering: little endian

Using binary dir /usr/lib/yamagi-quake2/baseq2/ to fetch paks
Added packfile '/usr/lib/yamagi-quake2/baseq2/pak0.pak' (3307 files).
Using '/home/lmello/.yq2/baseq2' for writing.
execing default.cfg
couldn't exec yq2.cfg
couldn't exec config.cfg
couldn't exec autoexec.cfg
Console initialized.

------- sound initialization -------
...opening OpenAL device:ok
...creating OpenAL context: ok
...making context current: ok

OpenAL settings:
AL_VENDOR: OpenAL Community
AL_RENDERER: OpenAL Soft
AL_VERSION: 1.1 ALSOFT 1.18.0
AL_EXTENSIONS: AL_EXT_ALAW AL_EXT_BFORMAT AL_EXT_DOUBLE AL_EXT_EXPONENT_DISTANCE AL_EXT_FLOAT32 AL_EXT_IMA4 AL_EXT_LINEAR_DISTANCE AL_EXT_MCFORMATS AL_EXT_MULAW AL_EXT_MULAW_BFORMAT AL_EXT_MULAW_MCFORMATS AL_EXT_OFFSET AL_EXT_source_distance_model AL_EXT_SOURCE_RADIUS AL_EXT_STEREO_ANGLES AL_LOKI_quadriphonic AL_SOFT_block_alignment AL_SOFT_deferred_updates AL_SOFT_direct_channels AL_SOFT_gain_clamp_ex AL_SOFT_loop_points AL_SOFT_MSADPCM AL_SOFT_source_latency AL_SOFT_source_length AL_SOFT_source_resampler AL_SOFT_source_spatialize

Available OpenAL devices:
- Built-in Audio Analog Stereo
- HDA ATI HDMI Digital Stereo (HDMI)

Current OpenAL device:
- OpenAL Soft

Number of OpenAL sources: 31

Starting Ogg Vorbis.
OGG_LoadPlaylist: could not open playlist: No such file or directory.
20 Ogg Vorbis files found.
Sound sampling rate: 0
------------------------------------

----- refresher initialization -----
LoadLibrary(ref_gl1.so)
Sys_LoadLibrary failed: /usr/lib/yamagi-quake2/ref_gl1.so: undefined symbol: glVertexPointer
Shutting down Ogg Vorbis.
Shutting down OpenAL.
Cmd_RemoveCommand: force_centerview not added
Cmd_RemoveCommand: +mlook not added
Cmd_RemoveCommand: -mlook not added
Shutting down input.
recursive shutdown
Error: Loading ref_gl1.so as renderer lib failed!
```

**Comment by Yamagi** [comment]
I can't reproduce that on my Arch system. Can you tell us what libGL provider do you have installed and what's you GPU?

**Comment by sljunkie** [comment]
I have libGL from libglvnd and an AMD RX 480; Mesa 17.1.5

```
Extended renderer info (GLX_MESA_query_renderer):
    Vendor: X.Org (0x1002)
    Device: AMD POLARIS10 (DRM 3.8.0 / 4.9.39-1-MANJARO, LLVM 4.0.1) (0x67df)
    Version: 17.1.5
    Accelerated: yes
    Video memory: 4068MB
    Unified memory: no
    Preferred profile: core (0x1)
    Max core profile version: 4.5
    Max compat profile version: 3.0
    Max GLES1 profile version: 1.1
    Max GLES[23] profile version: 3.1
```

**Comment by Yamagi** [comment]
*hmpf* This is caused by a bug in our CMakeLists.txt. We didn't correctly link libGL into the renderers. I think that this is solved in commit 0be75560fbf9b8f0f7edc9a99f8c949fef26c70e that was merged this morning.
